03week/ticTacToe.js

```diff
@@ -1,5 +1,106 @@
 'use strict';
 
+//MY CODE PLAN!
+
+//validate the user input - it must be equal to either 0, 1, or 2
+const validInputArr = [0, 1, 2]
+
+const checkInput = (input) => {
+  return validInputArr.indexOf(input) != -1
+}
+
+const validate = (row, column) => {
+  return checkInput(row) && checkInput(column)
+}
+
+//check to see if the selected row and column are occupied or not - function isPositionOpen = if index[row][column] != ''
+const isPositionOpen = (row, column) => {
+  return board[row][column] == ' '
+}
+
+//take user input and turn it into either an X or an O on the board - function placeMove = board[1][1] = 'X'
+const placeMoveX = (row, column) => {
+  board[row][column] = 'X';
+  console.log(board);
+}
+
+const placeMoveO = (row, column) => {
+  board[row][column] = 'O';
+  console.log(board);
```

Comment:
Nit pick. Notice these 2 functions do the exact same thing. Think about a way to do these 2 functions with one function.